viam-orb-slam3_test.go

```diff
@@ -165,23 +140,6 @@ func setupDeps(attr *slamConfig.AttrConfig) registry.Dependencies {
 		cam := &inject.Camera{}
 		switch sensor {
 		case "good_lidar":
-			cam.NextPointCloudFunc = func(ctx context.Context) (pointcloud.PointCloud, error) {
-				return pointcloud.New(), nil
-			}
-			cam.StreamFunc = func(ctx context.Context, errHandlers ...gostream.ErrorHandler) (gostream.VideoStream, error) {
-				return nil, errors.New("lidar not camera")
-			}
-			cam.ProjectorFunc = func(ctx context.Context) (transform.Projector, error) {
-				return nil, transform.NewNoIntrinsicsError("")
-			}
-			cam.PropertiesFunc = func(ctx context.Context) (camera.Properties, error) {
-				return camera.Properties{}, nil
-			}
-			deps[camera.Named(sensor)] = cam
-		case "bad_lidar":
```

Comment:
Mind explaining why we deleted case "bad_lidar" but left "good_lidar"?

Reply:
See https://github.com/viamrobotics/viam-orb-slam3/pull/19#discussion_r1138825390 and https://github.com/viamrobotics/viam-orb-slam3/pull/19#discussion_r1137603062. Basically, we have a test that proves that we fail on invalid input types to this algo and we dont want to get rid of it

Reply:
I see, thanks! Would it make sense to call it `invalid_sensor`?

Reply:
Agreed. Thank you for asking about this.